### PR TITLE
Feat:Added rating component

### PR DESCRIPTION
**Pull request by borkarsaish65**
image

You can check a workable demo by clicking on below link:

https://voluble-pika-55d3b0.netlify.app/component/rating-component/rating.html

### CODE REVIEW

**Review by RahullRawat — APPROVED** [review]

File: component/rating-component/rating.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    margin:0 3px;
+    font-size: 1.565rem;
+}
+
+.rating-div > input:checked ~ label {
+    color: #ffc700;    
+}
+.rating-div:not(:checked) > label:hover,
+.rating-div:not(:checked) > label:hover ~ label {
+    color: #deb217;
```
Comment (DevRaahul) [inline component/rating-component/rating.css:25]: if it is a component in the UI library then you can use color constant.

File: component/rating-component/rating.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+   display: none;
+}
+.rating-div:not(:checked) > label {
+    float:right;
+    margin:0 3px;
+    font-size: 1.565rem;
+}
+
+.rating-div > input:checked ~ label {
+    color: #ffc700;
```
Comment (DevRaahul) [inline component/rating-component/rating.css:21]: if it is a component in the UI library then you can use color constant.

File: component/rating-component/rating.css [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+.rating-div:not(:checked) > label:hover,
+.rating-div:not(:checked) > label:hover ~ label {
+    color: #deb217;  
+}
+.rating-div > input:checked + label:hover,
+.rating-div > input:checked + label:hover ~ label,
+.rating-div > input:checked ~ label:hover,
+.rating-div > input:checked ~ label:hover ~ label,
+.rating-div > label:hover ~ input:checked ~ label {
+    color: gold;
```
Comment (DevRaahul) [inline component/rating-component/rating.css:32]: if it is a component in the UI library then you can use color constant.

File: component/rating-component/rating.html [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+&lt;label for="star1" title="text" class="fa fa-star"&gt;&lt;/label&gt;
+&lt;/div&gt;
+&lt;/div&gt;
+                </code></pre>
+            </div>
+
+        </div>
+
+
+
```
Comment (DevRaahul) [inline component/rating-component/rating.html:115]: Use Prettier extension for formatting

File: component/rating-component/rating.html [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+
+        <script>
+
+        </script>
+</body>
+
+</html>
```
Comment (AnkurChunekar) [inline component/rating-component/rating.html:122]: add a last line at the end of this file

**Review by AnkurChunekar — APPROVED** [review]

File: component/rating-component/rating.html (hunk 5 above)
Comment (borkarsaish65) [reply component/rating-component/rating.html:122]: Yes sir will do that. Thanks.

File: component/rating-component/rating.html (hunk 4 above)
Comment (borkarsaish65) [reply component/rating-component/rating.html:115]: Yes will do. Thanks for such extensive review.

File: component/rating-component/rating.css (hunk 3 above)
Comment (borkarsaish65) [reply component/rating-component/rating.css:32]: Yess will do Rahul.

**Pull request merged**